content/docs/projects/README.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Projects
+description: 'Satellite projects of cert-manager'
+---
+
+The cert-manager project has a number of [satellite
+projects](https://github.com/cert-manager) that extend the functionality, and
+compliment the core cert-manager feature-set. Below is a list of projects which
+might help you with particular use-cases, or bolster your cluster's security and
+compliance.
```

Comment:
suggestion: this doesn't read quite right and there's a typo (compliment -> complement):

```suggestion
The cert-manager project has a number of [satellite
projects](https://github.com/cert-manager) that extend the project's functionality and
complement the core cert-manager feature-set.

These tools help with security, compliance and control.
```